test_runner/fixtures/neon_fixtures.py

```diff
@@ -904,6 +905,7 @@ def __init__(self, config: NeonEnvBuilder):
         # generate initial tenant ID here instead of letting 'neon init' generate it,
         # so that we don't need to dig it out of the config file afterwards.
         self.initial_tenant = config.initial_tenant
+        self.initial_timeline: None | TimelineId = None
```

Comment:
I think we usually use it with `Optional`:

```suggestion
        self.initial_timeline: Optional[TimelineId] = None
```


Note: I think we can enforce the style of this with `ruff` (https://beta.ruff.rs/docs/rules/#pyupgrade-up), but it'll change a lot of code, so I'd do it in a separate PR

Reply:
Aah yes, apologies on this. So obvious when I see your suggestion, but honestly didn't even remember python types had Optional. Which ... I think is surprising for others as well, since the gradual typing fully supports this `None | something_else`..

I'll commit this in anyways, since I guess it's more typed-pythonic.